receiver/vcenterreceiver/metadata.yaml

```
@@ -470,6 +470,16 @@ metrics:
     extended_documentation: As measured over the most recent 20s interval.
     warnings:
       if_enabled_not_set: "this metric will be enabled by default starting in release v0.102.0"
+  vcenter.vm.network.packet.drop.rate:
```

Naming is always a bit opinionated but typically periods used to delineate between namespaces, where the namespace refers to a thing - in this case, the packets. The drop rate is a measure relating to those packets but "drop" isn't really a thing which should have it's own namespace IMO

```suggestion
    vcenter.vm.network.packet.drop_rate:
```
```suggestion
  vcenter.vm.network.packet.drop_rate:
```

---

Talked about this offline. @StefanKurek pointed out that the collector uses `.rate` in many receivers but never `_rate`. I think some of the namespaces used elsewhere don't make sense due to this but for the sake of consistency I think we just leave it.